src/main/webapp/css/user_style.css

```diff
@@ -0,0 +1,83 @@
+.main {
+    text-align: center;
+    padding: 0px 10px;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+.column {
+    display: inline-block;
+    width: 45%;
+}
+
+.autocomplete {
+    position: relative;
+    display: inline-block;
+}
+
+input {
+    border: 1px solid transparent;
+    background-color: #f1f1f1;
+    padding: 10px;
+    font-size: 16px;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+input[type=text] {
+    background-color: #f1f1f1;
+    width: 100%;
+}
+
+input[type=submit] {
+    background-color: #4285F4;
+    color: #fff;
+}
+
+.autocomplete-items {
+    border: 1px solid #d4d4d4;
+    border-top: none;
+    z-index: 99;
+    top: 100%;
+    max-height: 300px;
+    overflow: scroll;
+    text-align: left;
+}
+
+.autocomplete-items div {
+    padding: 10px;
+    cursor: pointer;
+    border-bottom: 1px solid #d4d4d4;
+}
+
+.autocomplete-items div:hover {
+    background-color: #e9e9e9;
+}
+
+.autocomplete-active {
+    background-color: #4285F4;
+    color: #ffffff;
+}
+
+.setting {
+    margin-top: 20%;
+}
+
+.input-websites {
+    width: 100%;
+    display: block;
+    margin: auto;
+    border: 1px solid transparent;
+    background-color: #f1f1f1;
+    padding: 10px;
+    font-size: 16px;
+    font-family: Roboto, Arial, sans-serif;
+}
+
+.setting-submit {
+    display: block;
+    margin-top: 5%;
+    margin-left: 47.5%;
```

Comment:
Can you just center it instead of using percentage-based margins?

Reply:
In general the layout of this form should be revisited. Do you really want the select/input fields to be half the width of the screen? That's going to be way too large in general on most desktop/laptop devices. How about setting a max-width on these inputs, and designing your form's layout to have two centered rows?